<?xml version="1.0"?>
<launch>

    <node pkg="grasp_n_sorter" type="cloud_publisher_rviz.py" name="cloud_publisher_rviz" output="screen" />

    <node pkg="grasp_n_sorter" type="identify_object.py" name="identify_object" output="screen" />

    <arg name="frame_id_pc" default="camera_color_optical_frame"/>

    <node pkg="grasp_n_sorter" type="get_grasps_server.py" name="get_grasps_server" output="screen">
      <param name="frame_id_pc" value="$(arg frame_id_pc)"/>
    </node>


</launch>